src/app/pages/list/data-list/data-list.component.ts

```diff
@@ -1,45 +1,33 @@
-import { Component, Pipe, PipeTransform } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
     selector: 'app-data-list',
     templateUrl: './data-list.component.html',
     styleUrls: ['./data-list.component.scss'],
 })
 export class DataListComponent {
-    list = {
-        georgeWashington: 1789,
-        johnAdams: 1796,
-        thomasJefferson: 1800,
-        jamesMadison: 1808,
-        jamesMonroe: 1812,
-    };
+    list = [
+        {
+            name: 'George Washington',
+            year: 1789,
+        },
+        {
+            name: 'John Adams',
+            year: 1796,
+        },
+        {
+            name: 'Thomas Jefferson',
+            year: 1800,
+        },
+        {
+            name: 'James Madison',
+            year: 1808,
+        },
+        {
+            name: 'James Monroe',
+            year: 1812,
+        },
+    ];
 
     constructor() {}
 }
-
```

Comment:
Glad you got rid of these pipes, not sure why the data set was so complicated to begin with.